### PR TITLE
Veto and upgrade delay tweaks

### PR DESCRIPTION
**Pull request by Valdorff**

### CODE REVIEW

**Review by sckuzzle — APPROVED** [review]
looks good

**Comment by darrenlangley** [comment]
LGTM

File: RPIPs/RPIP-60.md [hunk 1]
```diff
@@ -28,16 +28,18 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 
 ### Upgrade Delay
 - All protocol upgrades SHALL have an `upgrade_delay` delay between when they are passed and when they are executed
-- The `upgrade_delay` SHALL be a pDAO setting and its initial value SHALL be 3 weeks
+- The `upgrade_delay` SHALL be a pDAO setting and its initial value SHALL be 1 week
 
 
 ### Security Council Veto
 - The security council SHALL have the power to veto contract upgrades (which are proposed by the oDAO)
+  - `rocketDAOProtocolSettingsSecurity.upgradeveto.quorum` SHALL be a pDAO setting and its initial value SHALL be 33%
```
Comment (LongForWisdom) [inline RPIPs/RPIP-60.md:36]: I agree we need to set this somewhere. Should it be here, or should it defer to as yet not existing Security Council RPIP? I'm happy to just have a rationale item or something saying its here for now because the other thing doesn't exist, and instruct RPIP editors to move / remove it in the future if deemed appropriate?

Comment (LongForWisdom) [reply RPIPs/RPIP-60.md:36]: Feel free to disregard, more of a stylistic / structuring thing than anything.

File: RPIPs/RPIP-60.md [hunk 2]
```diff
@@ -52,6 +54,7 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
...
 - The veto process can itself be modified by contract upgrades. However, if a corrupt proposal tries to change the veto process, that can be defended against by using the veto process.
 - The security council veto means that if pDAO governance were captured, protocol upgrades could be stopped at will (by installing a compromised security council). The oDAO would be unable to pass any contract upgrades (without the approval of the compromised security council), and the only way back would be to recapture pDAO governance. While this means there is a real tradeoff, this "inability to upgrade" is seen as less problematic than the "ability to pass any upgrade" that a rogue oDAO would have without the veto.
+- The quorum for this veto is lower than for security council proposals. This is intended to provide additional resilience to, eg, a rogue oDAO that attempts to bribe a security council into not vetoing a corrupt proposal (as fewer bribe-resistant members are needed). This sort of interaction doesn't exist for normal security council proposals, which is what leads to a different desired threshold.
```
Comment (LongForWisdom) [inline RPIPs/RPIP-60.md:57]:
```suggestion
- The quorum for this veto is lower than for security council proposals. This is intended to provide additional resilience to, eg, a rogue oDAO that attempts to bribe a security council into not vetoing a corrupt proposal (as fewer bribe-resistant members are needed).
```
I'm not sure that last statement is true? And it seems difficult to prove the negative. Maybe just remove? Stylistically its also providing rationale on something not within this RPIP (normal security council proposals)

Comment (LongForWisdom) [reply RPIPs/RPIP-60.md:57]: Also feel free to ignore.

Comment (Valdorff) [reply RPIPs/RPIP-60.md:57]: Happy to delete that line. Was trying to provide a reason for not using the existing `rocketDAOProtocolSettingsSecurity.members.quorum` setting

File: RPIPs/RPIP-60.md (hunk 1 above)
Comment (Valdorff) [reply RPIPs/RPIP-60.md:36]: I think it should be here. I'm not against reorganizing at some point if helpful (the majority of the SC stuff is RPIP-33)

File: RPIPs/RPIP-60.md (hunk 2 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-60.md:57]: Ah, okay. Maybe reword first sentence to better capture this? 'The quorum for this veto is lower than for security council proposals set in rocketDAOProtocolSettingsSecurity.members.quorum'
